src/java.base/share/classes/java/lang/foreign/SequenceLayout.java

```diff
@@ -70,15 +70,16 @@ public sealed interface SequenceLayout extends MemoryLayout permits SequenceLayo
      * Returns a sequence layout with the same element layout, alignment constraint and name as this sequence layout,
      * but with the specified element count.
      * @param elementCount the new element count.
-     * @return a sequence layout with the given element count.
-     * @throws IllegalArgumentException if {@code elementCount < 0}.
+     * @return a sequence layout with the same characteristics of this layout, but with the given element count.
+     * @throws IllegalArgumentException if {@code elementCount} is negative.
+     * @throws IllegalArgumentException if {@code elementLayout.bitSize() * elementCount} overflows.
```

Comment:
Shouldn't the exception descriptions be combined to avoid IAE being listed twice in the generated javadoc?

Reply:
I think @PaulSandoz  suggested this idiom in other places where the `@throws` clause was very hard to break up, so I thought that would have been preferrable than having a single para. But if there are strong opinions, I could tweak in different ways, perhaps with bullet lists (but that will require a lot of work).